src/main/java/sc/fiji/blind_experiment/UnblindFiles.java

```diff
@@ -0,0 +1,72 @@
+package sc.fiji.blind_experiment;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.Properties;
+
+import org.apache.commons.io.FilenameUtils;
+import org.scijava.command.Command;
+import org.scijava.log.LogService;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+import org.scijava.prefs.PrefService;
+
+/**
+ * Blind Files Plugin Command
+ * <p>
+ * This plugin unblinds a folder of files that have been previously blinded.
+ *
+ * @author Jan Eglinger
+ * @author Andrew Valente
+ */
+
+@Plugin(type = Command.class, menuPath = "Analyze>Blind Experiment>Unblind Files")
+public class UnblindFiles implements Command {
+	
+	@Parameter()
+	PrefService prefs;
+	
+	@Parameter()
+	LogService log;
+	
+	@Override
+	public void run() {
+		// Unblind the files using the saved props file
+		File propsFile = new File(prefs.get("blindPropsPath"));
```

Comment:
This leaves no possibility to unmask files/results that were masked earlier in the past. If you masked several folders, only the last one will be saved in the prefs, no?
Let's keep the directory as an input parameter as I had it in my original implementation. We can still use a `DynamicCommand` with an `initializer` method to pre-fill the path with the one saved in the prefs.